functions-methods.js

```diff
@@ -33,4 +56,28 @@
 // checkEmailValidity("tessmellink@novi.nl") geeft true - want @ en punt op de juiste plek
 // checkEmailValidity("n.eekenanovi.nl") geeft false - want geen @
 // checkEmailValidity("n.eeken@novinl.") geeft false - want de punt mag niet als laatst
-// checkEmailValidity("tessmellink@novi,nl") geeft false - want er staat een komma in
\ No newline at end of file
+// checkEmailValidity("tessmellink@novi,nl") geeft false - want er staat een komma in
+console.log("opdracht 3");
+var checkEmailValidity = function (email) {
```

Comment:
var nooit gebruiken svp, die is namelijk deprecated. Hier heb je een let of een const voor!!!